client/data/authorizations/test/selectors.test.ts

```diff
@@ -0,0 +1,146 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	getAuthorization,
+	getAuthorizations,
+	getAuthorizationsSummary,
+} from '../selectors';
+import { getResourceId } from 'utils/data';
+import { RiskLevel } from 'wcpay/types/authorizations';
+
+const emptyState = { authorizations: {} };
+
+describe( 'Authorizations selector', () => {
+	const mockQuery = { paged: '2', perPage: '50' };
+	const mockAuthorizations = [
+		{
+			authorization_id: 'id_1661935621753_995',
+			authorized_on: 'Aug 31, 2022 / 8:47AM',
+			capture_by: 'Sep 7, 2022 / 8:47AM',
+			order: {
+				number: 254,
+				customer_url: 'https://doggo.com',
+				url: 'https://doggo.com',
+			},
+			risk_level: 'elevated' as RiskLevel,
+			amount: 4654,
+			customer_email: 'good_boy@doge.com',
+			customer_country: 'Kingdom of Dogs',
+			customer_name: 'Good boy',
+			payment_intent_id: 'pi_3Lcm2iQsDOQXPzI102uKS0FD',
+		},
+		{
+			authorization_id: 'id_1661935621753_107',
+			authorized_on: 'Aug 31, 2022 / 8:47AM',
+			capture_by: 'Sep 7, 2022 / 8:47AM',
+			order: {
+				number: 254,
+				customer_url: 'https://doggo.com',
+				url: 'https://doggo.com',
+			},
+			risk_level: 'normal' as RiskLevel,
+			amount: 4906,
+			customer_email: 'good_boy@doge.com',
+			customer_country: 'Kingdom of Dogs',
+			customer_name: 'Good boy',
+			payment_intent_id: 'pi_3Lcm2iQsDOQXPzI102uKS0FD',
+		},
+	];
+
+	const filledSuccessState = {
+		authorizations: {
+			byId: {},
+			[ getResourceId( mockQuery ) ]: {
+				data: mockAuthorizations,
+			},
+			summary: {},
+		},
+	};
+
+	test( 'Returns empty disputes list when disputes list is empty', () => {
```

Comment:
We may have a typo here - `disputes` in place of `authorizations`.

Reply:
Oops thanks. Fixed in [d4431d4](https://github.com/Automattic/woocommerce-payments/pull/4676/commits/d4431d4d6c8ae4d367f5ab2a69125d17bdcd4eb3)